libs/dh/shared/feature-flags/src/lib/feature-flags.spec.ts

```diff
@@ -14,15 +14,15 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { differenceInDays, parse } from 'date-fns';
+import { dayjs } from '@energinet-datahub/watt/date';
 import { DhFeatureFlag, DhFeatureFlags, dhFeatureFlagsConfig } from './feature-flags';
 
 const maxAgeOfDays = 62;
 
 const featureFlagCases = Object.keys(dhFeatureFlagsConfig).map((featureFlag) => {
   const created = (dhFeatureFlagsConfig[featureFlag as DhFeatureFlags] as DhFeatureFlag).created;
-  const parsedDate = parse(created, 'dd-MM-yyyy', new Date());
-  const diffInDays = differenceInDays(new Date(), parsedDate);
+  const parsedDate = dayjs(created, 'dd-MM-yyyy');
```

Comment:
issue: format pattern is not valid.


```suggestion
  const parsedDate = dayjs(created, 'DD-MM-YYYY');
```